reana_db/utils.py

```diff
@@ -268,7 +269,16 @@ def update_users_disk_quota(
                 user_id=u.id_, resource_id=disk_resource.id_
             ).first()
             if bytes_to_sum is not None:
-                user_resource_quota.quota_used += bytes_to_sum
+                updated_quota_usage = user_resource_quota.quota_used + bytes_to_sum
+                if updated_quota_usage < 0:
+                    logging.warning(
+                        f"Disk quota of user {u.id_} would become negative, "
```

Comment:
Cosmetics: the current WARNING message says:

```
Disk quota of user c95fbde3-c947-4a28-823e-f326faab0027 would become negative, setting to zero. (0 [original] + -188416 [delta] = -188416)
```

This might be perhaps misinterpreted with respect to what is the final state. (Note two zeros.) 

What about using a "chronological order" to clarify what is the new state, for example:

```
Disk quota consumption of user c95fbde3-c947-4a28-823e-f326faab0027 would become negative: 0 [original usage] + -188416 [delta] -> -188416 [new usage]. Setting the new usage to zero.
```

Or hide the details and just say:

```
Disk quota consumption of user c95fbde3-c947-4a28-823e-f326faab0027 would become negative. Setting to zero.
```